components/src/core/components/InputField/InputField.vue

```diff
@@ -70,9 +73,9 @@ export default defineComponent({
     labelIcon: {
       type: String,
     },
-    // this prop only applicable for `type`='file'
-    buttonLabel: {
```

Comment:
After removing this file input working fine. right?
Any reason why this removed

Reply:
yes this is working. removed this because we have added `v-bind="$attrs"` in component, no need to explicitly mention props unless we want to target surrounding component.